utils/docker/Dockerfile.centos.8

```diff
@@ -243,7 +243,18 @@ RUN echo -e "<settings>\n\
 </settings>" > /home/daos/.m2/settings.xml
 
 ARG DAOS_JAVA_BUILD=$DAOS_BUILD
+ARG BASE_DISTRO
```

Comment:
You already have ARG BASE_DISTRO above, right at the start so don't need this line.

Reply:
As tested, I cannot read any value from BASE_DISTRO if I don't redefine it again. I don't know why yet. Any idea?

Reply:
No, as far as I know it's just a regular arg.

Reply:
As tested again, I need to redefine this arg to make fedora build work.